components/pedigree/resources/src/main/resources/pedigree/view/person.js

```diff
@@ -1579,6 +1579,8 @@ define([
                     //          {id: 'ENSG00000160688', gene: 'FLAD1', status: 'solved'},
                     //          {id: 'custom', 'gene': 'custom', status: 'candidate'} ]
                     this._setGenes(info.genes);
+                } else {
+                    this._setGenes([]);
```

Comment:
However, more generic code (that didn't just go through each and every one of many fields to do the same thing, but somehow triggered the update on all) would probably not have had the bug in the first place.